module/Application/config/module.config.php

```diff
@@ -67,4 +128,19 @@
             __DIR__ . '/../view',
         ],
     ],
+
+    'doctrine' => [
+        'driver' => [
+            'application_entity' => [
```

Comment:
In a subsequent PR, we should consider renaming `Application`. Opened #8 for this.